sprox/sa/widgetselector.py

```diff
@@ -41,6 +41,7 @@ class SAWidgetSelector(WidgetSelector):
     DateTime: SproxCalendarDateTimePicker,
     Date:     SproxCalendarDatePicker,
     Time:     SproxTimePicker,
+    BINARY:   FileField,
```

Comment:
I think you just want to add `_Binary`here. In the end `BINARY`  is just a subclass of `_Binary` and we certainly don't want to list all aliases of SQLAlchemy types as there are tons of them https://github.com/zzzeek/sqlalchemy/blob/master/lib/sqlalchemy/types.py#L61-L115

The selector should already deal with subclassing -> https://github.com/TurboGears/sprox/blob/master/sprox/sa/widgetselector.py#L82-L85